apps/dashboard/src/components/activity/activity-job-item.tsx

```diff
@@ -51,7 +51,7 @@ export function ActivityJobItem({ job, isFirst, isLast }: ActivityJobItemProps)
                 {getJobIcon(job.type)}
               </div>
             </div>
-            <span className="text-foreground-950 text-xs capitalize">{formatJobType(job.type)}</span>
+            <span className="text-foreground-950 text-xs capitalize">{job?.step?.name || formatJobType(job.type)}</span>
```

Comment:
show the job step name instead of the job type

Reply:
Can you please point me to the place for this span?

Reply:
![CleanShot 2024-12-27 at 10 44 14@2x](https://github.com/user-attachments/assets/0d846231-9f6f-4752-bfab-cfd2266e5e0b)